x-pack/solutions/security/plugins/security_solution/public/data_view_picker/components/data_view_picker/index.test.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { DataViewPicker } from '.';
+import { useDataView } from '../../hooks/use_data_view';
+import { DEFAULT_SECURITY_SOLUTION_DATA_VIEW_ID, DataViewPickerScopeName } from '../../constants';
+import { shared } from '../../redux/slices';
+import { useDispatch } from 'react-redux';
+import { useKibana } from '../../../common/lib/kibana';
+import { DataView } from '@kbn/data-views-plugin/common';
+import { FieldFormatsRegistry } from '@kbn/field-formats-plugin/common';
+import { TestProviders } from '../../../common/mock/test_providers';
+import { useSelectDataView } from '../../hooks/use_select_data_view';
+
+jest.mock('../../hooks/use_data_view', () => ({
+  useDataView: jest.fn(),
+}));
+
+jest.mock('../../hooks/use_select_data_view', () => ({
+  useSelectDataView: jest.fn().mockReturnValue(jest.fn()),
+}));
+
+jest.mock('react-redux', () => {
+  return {
+    ...jest.requireActual('react-redux'),
+    useDispatch: jest.fn(),
+  };
+});
+
+jest.mock('../../../common/lib/kibana', () => ({
+  useKibana: jest.fn(),
+}));
+
+jest.mock('@kbn/unified-search-plugin/public', () => ({
+  ...jest.requireActual('@kbn/unified-search-plugin/public'),
+  DataViewPicker: jest.fn((props) => (
```

Comment:
Could we just export a mock from unified search here since all the DataViewPicker does is call ChangeDataView? `src/platform/plugins/shared/unified_search/public/dataview_picker/change_dataview.test.tsx`?

Reply:
do you mean I should add this mock to the unified search plugin?